### PR TITLE
fixes to widths and adding time of day backgrounds to header

### PR DESCRIPTION
**Pull request by natedsaint**
## SUMMARY ##

The main board now slightly resizes, but mostly stays centered in big widths and doesn't collapse in small ones. I'll work on a responsive size next but this keeps it from breaking.

To see a demo, check out my hosted version: http://www.nathanstpierre.com:8080/

### CODE REVIEW

**Comment by natedsaint** [comment]
[Fix responsive breakpoint that is screwing up board in narrow view](https://trello.com/c/5WOHrF94/18-fix-responsive-breakpoint-that-is-screwing-up-board-in-narrow-view)

**Comment by natedsaint** [comment]
[Update header to change based on time of day (maybe randomly swap blue and red dude)](https://trello.com/c/A2oHqK2F/16-update-header-to-change-based-on-time-of-day-maybe-randomly-swap-blue-and-red-dude)

**Comment by Asuza** [comment]
@natedsaint, it looks like the team info shrunk just enough to force the ID badges to be on separate lines from the user name, and the "team diamond" count is on two lines instead of one as well. Was that intentional?

Text wrapping

**Comment by natedsaint** [comment]
The top line no, but I thought the player name/marker/hp was intentional since that change happened before I did anything (from jsfight.club right now: ) 
screen shot 2016-11-02 at 8 59 25 am

**Comment by natedsaint** [comment]
Oh I see, you meant the badge itself. Well if the name is long enough, that'll still happen: what should we do in those cases? 
I can definitely make this longer. I think maybe breakpoints with different font sizes will make things work in the future, but I just want this not to be broken for anyone even if they have to scroll until we get that done.

File: public/js/stylehelpers/freelancer.js [hunk 1]
```diff
@@ -72,7 +73,7 @@ var animateObjects = function() {
       $hero = $(".parallax-hero.red");
   
   $parallax.css({
-    "background-position":"50% -"+ (( scrollTop / 4 ) + 500 )+ "px "
+    "background-position":"50% -"+ (( scrollTop / 4 ) + 500 )+ "px ",
```
Comment (natedsaint) [inline public/js/stylehelpers/freelancer.js]: hmm. Interesting. I'd added the background-image here but then realized how bad of performance that would be so I put it in the stylesheet. I'll remove this, but this makes me want to look at the linting settings.

**Comment by Asuza** [comment]
I think it's fine if it does break... it just seems like it's breaking a lot more often when I last looked at the breakpoint changes. If that has to happen to prevent the team info lists from overlapping, then it is what it is. We can fix it later if that's the case.

File: public/js/stylehelpers/freelancer.js (hunk 1 above)
Comment (Asuza) [inline public/js/stylehelpers/freelancer.js]: Feel free to look at/review the linting settings. They're definitely causing goofiness in places.

**Comment by natedsaint** [comment]
got rid of stretch: we should circle back and make this responsive but for now I think it's fine.

**Pull request merged**